cytotable/convert.py

```diff
@@ -133,10 +133,20 @@ def _prepend_column_name(
         #   column_name: 'AreaShape_Area'
         #   updated_column_name: 'Cells_AreaShape_Area'
         if column_name not in identifying_columns and not column_name.startswith(
-            source_group_name_stem
+            source_group_name_stem.capitalize()
         ):
             updated_column_names.append(f"{source_group_name_stem}_{column_name}")
-
+        # if-condition for prepending 'Metadata_' to column name
+        # where colname is an identifying column
+        # and where the column is already prepended with source_group_name_stem
+        # for example:
+        #   source_group_name_stem: 'Cells'
+        #   column_name: 'Cells_Number_Object_Number'
+        #   updated_column_name: 'Metadata_Cells_Number_Object_Number'
```

Comment:
Will this prepend `Metadata` to columns that shouldn't have this prefix? E.g. will `Cells_AreaShape` become `Metadata_Cells_AreaShape`? We want to avoid this.

Reply:
the documentation is already stellar, but if my concern is not applicable, please add a brief note describing this.

Reply:
Thanks! The key with this if branch is that the column name must be an "identifying column" which is provided upfront as a parameter to `convert` via `identifying_columns`. In the example you provided, `Cells_AreaShape` would only ever be renamed that way if it was specified as an identifying column by a developer. Configuration preset examples can be found under the reused key name `CONFIG_IDENTIFYING_COLUMNS` under [`presets.config`](https://github.com/cytomining/CytoTable/blob/main/cytotable/presets.py). You're right in mentioning I should be more upfront about this in the documentation, I'll see about highlighting this better.